vignettes/User_Roles_and_Privileges.Rmd

```diff
@@ -305,13 +305,21 @@ A user with the `delete_package` privilege can remove packages from consideratio
 
 ### Overall Commentator
 
-A user with the `overall_comment` privilege can add/edit the overall comment and summary of a package.
+A user with the `overall_comment` privilege can add/edit the overall comment and summary of a package. Unlike maintenance or community usage metrics, each package can only have one overall comment and summary. A user with the `overall_comment` privilege can provide the overall recommendation for the package decision and provide the summary of what the package offers.
+
```

Comment:
Add a <br> here to create a little spacing between the paragraph and the image. But if you do that, you may want to do it preceding each image on this page/ vignette.

Reply:
I added a <br> before and after images.